docs/develop/worker-performance.mdx

```diff
@@ -36,10 +37,176 @@ The Workflow Cache is created and shared between all the workers. It's designed
+  Each Nexus Task contains the context required by the target Workflow Definition.
+
+Each Task Queue type provides unaggregated metrics.
+
+### Task Queue metrics {#task-queue-metrics-list}
+
+The Temporal Service reports information separately for each Task Queue type (not aggregated).
+Use the following metrics to retrieve detailed information about Task Queue health and performance.
+Available metrics include:
+
+- [`ApproximateBacklogCount`](#ApproximateBacklogCount)
+- [`ApproximateBacklogAge`](#ApproximateBacklogCount)
+- [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate)
+- [`BacklogIncreaseRate`](#BacklogIncreaseRate) (derived from [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate))
+
+#### `ApproximateBacklogCount` {#ApproximateBacklogCount}
+
+Represents the approximate count of Tasks currently backlogged in this Task Queue.
+The number may include expired Tasks as well as active Tasks, but it will eventually converge to the correct count over time.
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide partial information due to performance optimizations.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values for this metric.
+Since Tasks remain valid for only a few seconds in Sticky Queues, this inaccuracy diminishes over time, especially as the backlog grows.
+:::
+
+
+#### `ApproximateBacklogAge` {#ApproximateBacklogAge}
+
+Returns the approximate age of the oldest Task in the backlog.
+The age is based on the creation time of the Task at the head of the queue.
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide partial information due to performance optimizations.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values.
+Since Tasks remain valid for only a few seconds in Sticky Queues, this inaccuracy diminishes over time, especially when the backlog is older than a few seconds.
+
+:::
+
+#### `TasksAddRate` and `TasksDispatchRate` {#TasksAddRate-and-TasksDispatchRate}
+
+Reports the approximate Tasks-per-second added to or dispatched from a Task Queue.
+This rate is averaged over the most recent 30-second time interval.
+The calculations include Tasks that were added to or dispatched from the backlog as well as Tasks that were immediately dispatched and bypassed the backlog (sync-matched).
+
+:::info Metric Accuracy
+
+The actual Task delivery count may be significantly higher than the number reported by these metrics:
+
+- Tasks using Eager dispatch do not pass through Task Queues.
```

Comment:
Maybe move this sentence to after the Eager dispatch definition (so swap lines 124 and 125) to avoid fwd reference

Reply:
Will do that